pool/rate_limiting_test.go

```diff
@@ -15,35 +15,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSimpleRateLimitingConstructor(t *testing.T) {
-	var (
-		runner amboy.Runner
-		err    error
-	)
-
-	assert := assert.New(t)
-	queue := &QueueTester{
-		toProcess: make(chan amboy.Job),
-		storage:   make(map[string]amboy.Job),
-	}
-
-	runner, err = NewSimpleRateLimitedWorkers(1, time.Nanosecond, nil)
-	assert.Nil(runner)
-	assert.Error(err)
-	assert.Contains(err.Error(), "less than a millisecond")
-	assert.Contains(err.Error(), "nil queue")
-
-	runner, err = NewSimpleRateLimitedWorkers(0, time.Millisecond, nil)
-	assert.Nil(runner)
-	assert.Error(err)
-	assert.Contains(err.Error(), "pool size less than 1")
-	assert.Contains(err.Error(), "nil queue")
-
-	runner, err = NewSimpleRateLimitedWorkers(10, 10*time.Millisecond, queue)
-	assert.NoError(err)
-	assert.NotNil(runner)
-}
-
 func TestAverageRateLimitingConstructor(t *testing.T) {
```

Comment:
Maybe this file could be renamed to `rate_limiting_average_test.go` or something?

Reply:
I did the inverse and renamed the implementation file to `pool/rate_limiting.go` since we don't have an alternative rate limited pool anymore.